javascript/core/scope/closures.md

```diff
@@ -18,6 +18,11 @@ standards:
   javascript.functions.5: 10
 
 
+aspects:
+  - deep
```

Comment:
I would remove `deep` and add `introduction` and here's why. This is often one of the first times a programmer is going to deal with the concept of scope, whether they realize it or not. So this is something we want to share with them early on, so they have that exposure.